doc-content/enterprise-only/openshift/hacep-server-maven-proc.adoc

```diff
@@ -0,0 +1,132 @@
+[id='hacep-server-maven-proc']
+= Implementing the HA CEP server with a Maven repository for updating the KJAR service
+
+You can implement the HA CEP server that retrieves the KJAR service and all dependencies from a Maven repository that you provide. In this case, you can update the KJAR service at any time by updating it in the Maven repository and then making a call from the client code.
```

Comment:
To implement the HA CEP server that retrieves the KJAR service and all dependencies from a Maven repository that you provide, update the KJAR service at any time by updating it in the Maven repository and then making a call from the client code.